authorization/src/main/java/com/terabite/authorization/AuthorizationController.java

```diff
@@ -0,0 +1,39 @@
+package com.terabite.authorization;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthorizationController {
```

Comment:
Can we rename this to `LoginController` (rename file too) and have one file per endpoint? See joshua's comment for how to structure endpoint.

Reply:
I'll def rename. I'll need to read up more, but don't think we'll have multiple controllers since we're doing microservices. With the structure Joshua's recommending, it seems like we're gonna be using multiple services handled by one controller. [Structure](https://medium.com/learnwithnk/best-practices-in-spring-boot-project-structure-layers-of-microservice-versioning-in-api-cadf62bd3459)

Auth endpoints are also all gonna be working pretty similarly, so using multiple controllers just feels overkill.